kalite/topic_tools/__init__.py

```diff
@@ -266,16 +287,22 @@ def get_content_cache(force=False, annotate=False, language=None):
 
     if CONTENT is None:
         CONTENT = {}
+
+    if settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP and not force:
```

Comment:
The smallest change would be to restore this block to its original location. Or wait for me to open a PR targeting your branch here. :)